static/js/components/EmailCompositionDialog.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+import React from 'react';
+import Dialog from 'material-ui/Dialog';
+import Button from 'react-mdl/lib/Button';
+import R from 'ramda';
+import type { EMAIL_EDIT_STATE } from '../reducers/email';
+
+const createDialogActions = (close, send) => ([
+  <Button
+    type="button"
+    className="dialog-button cancel-button"
+    key="first"
+    onClick={close}
+  >
+    Cancel
+  </Button>,
+  <Button
+    type="button"
+    className="dialog-button save-button"
+    key="second"
+    onClick={send}
+  >
+    Send
+  </Button>
+]);
+
+const hitsCount = sk => R.isNil(sk) ? 0 : sk.getHitsCount();
```

Comment:
I'm fine with `s/sk/searchkit/g`, I had used `sk` to make the function body a little shorter.

I prefer Ramda because it's designed from the ground up for a functional style, so all the functions Ramda defines are autocurried, data-last, etc. It has a lot of handy methods for writing code in a functional style too, like `R.compose`, `R.map`, etc.

Lodash has a functional module (`lodash/fp`) which also does data-last, autocurried functions, but Ramda is more 'functional-nnative-, if that makes sense.

I've been using ramda where it makes sense to do so - I don't think we should really make a concerted effort to purge lodash, but I've been preferring writing in a more functional style lately and ramda is nice for that :)